### PR TITLE
Fix chair height

### PR DESCRIPTION
**Pull request by cebtenzzre**
Sitting height in a chair was changed (moved down by 0.2 blocks) to be more like other chair plugins for Bukkit and/or Spigot.
Insignificant change of code, somewhat significant change of player experience (regarding reports from a number of "testers" of this change on my server).

### CODE REVIEW

**Comment by me4502** [comment]
Does this still have the issue where players with high latency will sometimes fall inside the block client-side?

**Comment by Cebtenzzre** [comment]
I've never seen that happen before, but I have the tools to test such a condition.  I'll see if I can reproduce that behavior.

**Comment by me4502** [comment]
It used to happen a few years ago, it's hopefully been fixed since then, especially if other chair plugins have changed their behaviour to the lower height.

**Comment by Cebtenzzre** [comment]
Alright, so I did some testing (Linux has built in tools for simulating network latency) with 1000ms-2000ms ping or so, and found that _both_ height levels (original 1.5 and patched 1.2) cause players to fall inside/through chairs when dismounting.  AFAIK only the [Shevchik/Chairs](https://github.com/Shevchik/Chairs) Bukkit/Spigot plugin does anything effective about this, which is to save a player's position into a list before they sit, and after they unsit return them to their previous location from the list.  If I've read right and I remember correctly, some plugin(s) may teleport the player on top of / above the stairs on dismount, but in weird cases with low ceilings that _might_ cause suffocation issues.  Though the aforementioned plugin's method may also have issues should a void suddenly open where the player once was, that'd be much rarer and potentially correctable.  In any case, I like the previously linked plugin/repo's code so much I might look into porting it to Sponge myself, though it uses NMS so it may require a lot of rework.  Honestly, I've never written a plugin before, but I feel like I know what I'm doing from related experience and in-depth research.

TL;DR: My change seems to have no negative effects, but to fix existing problems have a look at what Shevchik's plugin does.

**Comment by me4502** [comment]
Hmm, thanks.

And CraftBook does put the player at the point where they were before they got into the chair - however it's an option in the config.
